spec/diagnose_spec.rb

```diff
@@ -373,15 +372,14 @@
         /    Sources:/,
         /      default: #{quoted "https://push.appsignal.com"}/,
         /      env:     #{quoted ENV["APPSIGNAL_PUSH_API_ENDPOINT"]}/,
-        /  environment: #{quoted "development"}/,
+        /  environment: #{quoted("development")}/,
```

Comment:
I removed the parenthesis in the method call to match with the majority of method calls in these matchers. Should we change them all to fit with one format as a standard?

Reply:
I just reverted the commit 🤷‍♂️ 
We can standardize it later. But this is why it's best to do that in a separate commit